### PR TITLE
More types

### PR DESCRIPTION
**Pull request by r-barnes**
Includes some alphabetizing of import lines.

### CODE REVIEW

**Comment by SteveDiamond** [comment]
Thanks! Some of the changes don't seem compatible with python 3.5.

**Comment by rileyjmurray** [comment]
@SteveDiamond the Python Foundation dropped support for 3.5 a while ago. We had a conversation about dropping support for 3.5 the last time I updated Travis/AppVeyor. The remaining mentions of 3.5 support in the docs should probably be updated to reflect no support for 3.5, and our GH Actions CI should probably not build for 3.5. Pinging @akshayka to make sure I'm not mis-remembering.

**Comment by SteveDiamond** [comment]
Hmm ok, I guess that's fine to drop 3.5. We had to abandon f-strings at one point because someone made a case for keeping 3.5.

File: cvxpy/settings.py [hunk 1]
```diff
@@ -164,7 +164,7 @@
 UNKNOWN = "UNKNOWN"
 
 # Numerical tolerances
-EIGVAL_TOL = 1e-10
+EIGVAL_TOL: float = 1e-10
```
Comment (SteveDiamond) [inline cvxpy/settings.py:167]: add types for all the values here?

Comment (SteveDiamond) [reply cvxpy/settings.py:167]: actually why would we do types for literals?

**Review by SteveDiamond — APPROVED** [review]
LGTM

**Comment by SteveDiamond** [comment]
Thanks @r-barnes ! It looks good to me. I was a little confused about types for constants.

File: cvxpy/settings.py (hunk 1 above)
Comment (r-barnes) [reply cvxpy/settings.py:167]: I've removed that now. I think Pyre at least prefers types for globals that are defined by function calls or similar, but I don't think it has a preference for literals. This was here as part of an attempt to fix something else.

**Pull request merged**